[2014-05-14 06:19:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:24:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:24:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:24:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:24:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:25:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:26:22] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:27:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:27:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:38:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:38:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:30] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:30] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:58] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:39:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:41:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:41:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:41:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:46:50] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:46:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:46:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:47:52] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:47:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:47:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:49:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:49:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:49:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:50:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:50:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:50:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:52:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:54:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:54:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:54:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:54:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:54:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:55:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:55:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:55:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:55:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:55:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:55:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:55:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:59:52] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:59:52] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 06:59:52] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:00:18] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:01:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:03:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:03:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:03:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:03:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:03:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:03:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:03:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:05:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-14 07:10:16] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/adobe02philwebservices/Sites/ace/app/routes.php(376): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('thank-you')
#2 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/adobe02philwebservices/Sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/adobe02philwebservices/Sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
